dd-sdk-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeRum.kt

```diff
@@ -111,11 +111,11 @@ internal class BridgeRum : DdRum {
         context: Map<String, Any?>
     ) {
         // TODO RUMM-899 override timestamp and stacktrace
-        GlobalRum.get().addError(
+        (GlobalRum.get() as? AdvancedRumMonitor)?.addErrorWithStacktrace(
```

Comment:
don't you need to remove the added `RumAttributes.ERROR_STACK` also ?

Reply:
Yes indeed